css/style.css

```diff
@@ -19,82 +19,85 @@ body {
 .header {
   height: 100vh; }
 
-* {
```

Comment:
Nitpick: This is the exact reason I was mentioning that it's best to not commit built resources to repository as it will also incur additional PR reviewing head-ache. 😆